<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTranslationPageClass</class>
 <widget class="QWidget" name="WTranslationPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WTranslationPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 16px &quot;微软雅黑&quot;;
	color#525252;
}	

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>471</width>
         <height>501</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>471</width>
         <height>501</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>422</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>Translation Setting</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>42</y>
          <width>420</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Translation type</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ck_tr_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>76</y>
          <width>175</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {width: 16px; height: 16px;}

QCheckBox::indicator:unchecked {border-image: url(:/QtTest/icon/Setting/soundbot/unsel.png);}

QCheckBox::indicator:checked {border-image: url(:/QtTest/icon/Setting/soundbot/sel.png);}
</string>
        </property>
        <property name="text">
         <string>General translation</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ck_tr_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>76</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {width: 16px; height: 16px;}

QCheckBox::indicator:unchecked {border-image: url(:/QtTest/icon/Setting/soundbot/unsel.png);}

QCheckBox::indicator:checked {border-image: url(:/QtTest/icon/Setting/soundbot/ckeck.png);}</string>
        </property>
        <property name="text">
         <string>High-precision translation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>122</y>
          <width>420</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Subtitle Display</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ck_sd_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>156</y>
          <width>175</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {width: 16px; height: 16px;}

QCheckBox::indicator:unchecked {border-image: url(:/QtTest/icon/Setting/soundbot/unsel.png);}

QCheckBox::indicator:checked {border-image: url(:/QtTest/icon/Setting/soundbot/ckeck.png);}</string>
        </property>
        <property name="text">
         <string>Translated subtitle</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ck_sd_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>156</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {width: 16px; height: 16px;}

QCheckBox::indicator:unchecked {border-image: url(:/QtTest/icon/Setting/soundbot/unsel.png);}

QCheckBox::indicator:checked {border-image: url(:/QtTest/icon/Setting/soundbot/ckeck.png);}</string>
        </property>
        <property name="text">
         <string>Bilingual subtitle</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Background Color</string>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_bk_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>236</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_bk_2">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>236</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>296</y>
          <width>420</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Translation Color</string>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_tl_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_tl_2">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>330</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_tl_3">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>330</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_tl_4">
        <property name="geometry">
         <rect>
          <x>168</x>
          <y>330</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>Original Color</string>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_og_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>428</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_og_2">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>428</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_og_3">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>428</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="WRadioBtn" name="ck_og_4">
        <property name="geometry">
         <rect>
          <x>168</x>
          <y>428</y>
          <width>36</width>
          <height>36</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WRadioBtn</class>
   <extends>QPushButton</extends>
   <header>model/WRadioBtn.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
